core/src/main/java/org/axonframework/config/Configuration.java

```diff
@@ -69,13 +70,19 @@ default EventStore eventStore() {
     }
 
     /**
-     * Returns the Event Processor Registry in this Configuration. If not set otherwise, the default one is {@link
-     * DefaultEventProcessorRegistry}.
+     * Returns the Event Processing Configuration in this Configuration.
      *
-     * @return the Event Processor Registry defined in this Configuration
+     * @return the Event Processing Configuration defined in this Configuration
      */
-    default EventProcessorRegistry eventProcessorRegistry() {
-        return getComponent(EventProcessorRegistry.class);
+    default EventProcessingConfiguration eventProcessingConfiguration() {
+        List<EventProcessingConfiguration> modules = getModules().stream()
+                                                                 .filter(m -> m.instance() instanceof EventProcessingConfiguration)
+                                                                 .map(m -> (EventProcessingConfiguration) m.instance())
+                                                                 .collect(Collectors.toList());
+        if (modules.size() > 1) {
+            throw new IllegalStateException("Found more than one EventProcessingConfiguration modules");
```

Comment:
Shouldn't this be an `AxonConfigurationException`?

Reply:
Yes, you are right!

Reply:
Woehoe ^^